charts/argo-cd/Chart.yaml

```diff
@@ -26,5 +26,10 @@ annotations:
     fingerprint: 2B8F22F57260EFA67BE1C5824B11F800CD9D2252
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
-    - kind: changed
-      description: Upgrade Argo CD to v2.9.2
+    - kind: added
+      description: [Docs] Added migration steps to README for Config Management Plugins settings available since 5.26.0
+      links:
```

Comment:
It'll be these links that are upsetting the CI. I'd just drop them.

Reply:
oh and, [docs] either needs the square brackets removed, or put the whole thing in quotes

Reply:
thanks, i'll check it again, just added quotes

Reply:
updated, looks like it was just the `[` and `]` :)

Reply:
Contributors tend to forget updating these fields in the future. We can test it for a while but I'd preffer to not add more artifacthub metadata to a release than documented in our contrib guideline

@jmeridth your opinion?

Reply:
Thank you @kzap for the update and thank you so much for contibuting the docs update ❤️